unified-codes/release-uc.sh

```diff
@@ -0,0 +1,58 @@
+### unified codes release steps                     ###
```

Comment:
I've been leaving this out of the codebase, but it is very useful. Time to include it?

Reply:
Nice!